src/javascript/binary/websocket_pages/user/account/authenticate.js

```diff
@@ -23,6 +32,236 @@ const Authenticate = (() => {
         });
     };
 
+    const init = () => {
+        // Setup accordion
+        $('.files').accordion({
+            heightStyle: 'content',
+            collapsible: true,
+            active     : false,
+        });
+        // Setup Date picker
+        const tomorrow = new Date();
+        tomorrow.setDate(tomorrow.getDate() + 1);
```

Comment:
can we use something like 

```
const tomorrow = window.time ? new Date(window.time.valueOf()) : new Date();
tomorrow.setDate(tomorrow.getDate() + 1);
```

similar to what we are using in duration file for trade page? as we already have time stored in window